geetools/ee_dictionary.py

```diff
@@ -80,3 +82,42 @@ def getMany(self, list: list | ee.List) -> ee.List:
                 d.geetools.getMany(["foo", "bar"]).getInfo()
         """
         return ee.List(list).map(lambda key: self._obj.get(key))
+
+    def toTable(self, valueType: ee.List | ee.Dictionary | Any = Any) -> ee.FeatureCollection:
+        """Convert a ee.Dictionary to a ee.FeatureCollection with no geometries (table).
```

Comment:
We now have full support of intersphinx so let's use it as much as we can: 

```suggestion
        """Convert a :py:class:`ee.Dictionary` to a :py:class:`ee.FeatureCollection` with no geometries (table).
```